lib/Parser/Constants.php

```diff
@@ -70,13 +70,17 @@ abstract class Constants
 
     const VK_ALL_DRIVERS = [
         "anv",
+        "lvp",
         "radv",
+        "tu",
         "v3dv",
     ];
 
     const VK_ALL_DRIVERS_VENDORS = [
         "Intel"     => [ "anv" ],
+        "Lavapipe"  => [ "lvp" ],
         "AMD"       => [ "radv" ],
+        "Turnip"    => [ "tu" ],
```

Comment:
From what I understood, turnip is a driver for Qualcomm hardware, so as for OpenGL, so I'd put `tu` under the `Qualcomm` vendor.
```suggestion
        "Qualcomm"    => [ "tu" ],
```